flexmark-coverage/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.acrolinx.flexmark</groupId>
+    <artifactId>flexmark-java</artifactId>
+    <version>0.70.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>flexmark-coverage</artifactId>
+  <description>An aggregator of all Jacoco test reports</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.acrolinx.flexmark</groupId>
+      <artifactId>flexmark</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.acrolinx.flexmark</groupId>
+      <artifactId>flexmark-core-test</artifactId>
```

Comment:
if I interpret https://www.eclemma.org/jacoco/trunk/doc/report-aggregate-mojo.html correctly, then we should still use `<scope>test</scope>` for all these test sub-modules like this one here?